backend-rust/src/migrations/m00010_fill_capital_bound_and_leverage_bound.rs

```diff
@@ -0,0 +1,90 @@
+//! Migration filling the newly added values `capital_bound` and
+//! `leverage_bound` to the database.
+
+use super::{SchemaVersion, Transaction};
+use anyhow::Context;
+use concordium_rust_sdk::{
+    types::PartsPerHundredThousands,
+    v2::{self, BlockIdentifier, ChainParameters},
+};
+use sqlx::Executor;
+
+/// Run database migration to fill the new columns `capital_bound` and
+/// `leverage_bound`.
+pub async fn run(
+    tx: &mut Transaction,
+    endpoints: &[v2::Endpoint],
+    next_schema_version: SchemaVersion,
```

Comment:
We only ever call it with `SchemaVersion::DelegatedStakeCap` so maybe make it a `const` in here

Reply:
My idea was to have all the schema transition visible in the `migration_to_next()` function, whereas if we hardcode constants into the migration fails, it is hard to keep track what schema outcome a particular migration has.

Reply:
okay, but it is a bit inconsistent now.
Anyway, this is not something blocking.